wp-content/themes/inf_theme/skin/assets/images/images.js

```diff
@@ -1,12 +1,5 @@
 import './logo.svg';
 
-// Browser icons.
-import './browsers/chrome.png';
-import './browsers/edge.png';
-import './browsers/firefox.png';
-import './browsers/opera.png';
-import './browsers/safari.png';
-
 // Favicons.
```

Comment:
Explore and test the option of using WordPress core `site-icon` functionality: https://codex.wordpress.org/Creating_a_Favicon